scaphandre/README.md

```diff
@@ -0,0 +1,68 @@
+# Agent Check: Scaphandre
+
+## Overview
+
+This check monitors [Scaphandre][1].
+
+## Setup
+
+### Installation
+
+To install the Scaphandre check on your host:
+
+
+1. Install the [developer toolkit]
+(https://docs.datadoghq.com/developers/integrations/python/)
+ on any machine.
+
+2. Run `ddev release build scaphandre` to build the package.
+
+3. [Download the Datadog Agent][2].
+
+4. Upload the build artifact to any host with an Agent and
+ run `datadog-agent integration install -w
+ path/to/scaphandre/dist/<ARTIFACT_NAME>.whl`.
```

Comment:
~~~suggestion
4. Upload the build artifact to any host with an Agent and run the following command:
    ```
    datadog-agent integration install -w path/to/scaphandre/dist/<ARTIFACT_NAME>.whl
    ```
~~~

Reply:
Done.